src/greeter/CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ message(STATUS "Building greeter for Qt ${QT_MAJOR_VERSION} as ${GREETER_TARGET}
 include_directories(
     "${CMAKE_SOURCE_DIR}/src/common"
     "${CMAKE_BINARY_DIR}/src/common"
-    "${LIBXCB_INCLUDE_DIR}"
+    ${LIBXCB_INCLUDE_DIR}
```

Comment:
Is this correct? If it can't include multiple dirs it should probably remain quoted

Reply:
The empty string fails when building with `BUILD_WITH_X11=OFF`. The variable itself should be either a string or a list. In both cases, it requires no quoting. But I'm not too deep into CMake...